test/HydraDelegation/VestedDelegation.test.ts

```diff
@@ -215,12 +215,72 @@ export function RunVestedDelegationTests(): void {
 
         await expect(tx)
           .to.emit(hydraDelegation, "PositionOpened")
-          .withArgs(vestManager.address, this.delegatedValidators[0], vestingDuration, amountToDelegate);
+          .withArgs(
+            vestManager.address,
+            this.delegatedValidators[0],
+            vestingDuration,
+            amountToDelegate.add(delegatedAmount)
+          );
 
         expect(await hydraDelegation.delegationOf(this.delegatedValidators[0], vestManager.address)).to.be.equal(
           delegatedAmount.add(amountToDelegate)
         );
       });
+
+      it("should open position, when there is no maturing reward (even if maturing period is not ended)", async function () {
+        const {
+          systemHydraChain,
+          hydraDelegation,
+          vestManager,
+          vestManagerOwner,
+          oldValidator,
+          newValidator,
+          hydraStaking,
+        } = await loadFixture(this.fixtures.swappedPositionFixture);
```

Comment:
Is that the right fixture that must be used?

Reply:
Yes, we need to recreate a position that is still maturing but has no rewards to claim (because we swapped)